src/components/loginform/LoginForm.scss

```diff
@@ -186,4 +186,72 @@ opacity: 1;
     padding-bottom: 10px;
     }
 
+    @media only screen and (max-width: 767px) {
```

Comment:
You can view this topic and work like this https://medium.com/codeartisan/breakpoints-and-media-queries-in-scss-46e8f551e2f2